src/main/java/com/example/project_spring_boot/security/SecurityConfig.java

```diff
@@ -39,6 +39,9 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
             .authorizeHttpRequests()
             .requestMatchers(HttpMethod.POST, SecurityConstants.REGISTRATION_PATH).permitAll()
             .requestMatchers(HttpMethod.GET, SecurityConstants.GETITEMS_PATH).permitAll()
+            .requestMatchers(HttpMethod.GET, SecurityConstants.GETBIDS_PATH).permitAll()
```

Comment:
GETITEMS_PATH are no longer needed to be permitALL, maybe we should remove it ?

Reply:
Yes good idea, removed it in new commit